guides/common/modules/proc_configuring-server-for-use-with-tftp.adoc

```diff
@@ -1,41 +1,53 @@
-[id="configuring-external-tftp_{context}"]
-= Configuring {ProductName} with external TFTP
+[id="configuring-server-for-use-with-tftp"]
+= Configuring {ProductName} for use with tftp
 
 You can configure {ProductName} with external TFTP services.
 
+
+.Prerequisites
+* You configured the TFTP server.
+* You shared the `/var/lib/tftpboot` on the TFTP server with NFS.
+
+
 .Procedure
+. Associate the TFTP service with the appropriate subnets.
+
+. For each affected {SmartProxy}, update the configuration of that {SmartProxy} in the {ProjectWebUI}. See xref:associating-the-tftp-service-with-a-subnet[].
```

Comment:
These 2 steps are really 1 step that you can only perform after you ran the installer command at the end of the procedure.

Reply:
I moved the "For each affected..." step to the end.

I removed the "associate" step because is covered in the section linked in the other step.